tests/terraform/Dockerfile.build

```diff
@@ -0,0 +1,27 @@
+FROM golang:alpine
+
+ARG TERRAFORM_VERSION=0.12.10
+ENV TERRAFORM_VERSION=$TERRAFORM_VERSION
+
+
+RUN apk update && \
+    apk upgrade --update-cache --available && \
+    apk add curl git jq bash openssh unzip gcc g++ make ca-certificates && \
+    curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
Why are we using dep instead of go mods?

Reply:
Updated. Thanks